pkg/machines/components/vmSnapshotsTab.jsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import moment from "moment";
+
+import cockpit from 'cockpit';
+import { vmId } from "../helpers.js";
+import { ListingTable } from "cockpit-components-table.jsx";
+import { Tooltip } from '@patternfly/react-core';
+import { InfoAltIcon } from '@patternfly/react-icons';
+
+import './vmSnapshotsTab.css';
+
+const _ = cockpit.gettext;
+
+function prettyTime(unixTime) {
+    const yesterday = _("Yesterday");
+    const today = _("Today");
+    moment.locale(cockpit.language, {
+        calendar : {
+            lastDay : `[${yesterday}] LT`,
+            sameDay : `[${today}] LT`,
+            sameElse : "L LT"
+        }
+    });
+
+    return moment(Number(unixTime) * 1000).calendar();
+}
+
+const VmSnapshotsTab = ({ vm }) => {
+    const id = vmId(vm.name);
+
+    const emptyCaption = (<div className="no-snapshots">
+        <h3>{_("No snapshots")}</h3>
+        {_("Previously taken snapshots allow you to revert to an earlier state if something goes wrong")}
+    </div>);
+
+    let detailMap = [
+        {
+            name: _("Creation Time"), value: (snap, snapId) => {
```

Comment:
Please use sentence case instead of headline case here.
https://www.patternfly.org/v4/design-guidelines/content/capitalization

Reply:
But apart from this nit, should be good to go!

Reply:
We currently use Title Case everywhere else in cockpit. IMHO we should keep that consistent for now, and then switch it around all in one go (issue #14285)

Reply:
Alright. So let's keep it like this and then make a big change in separate PR.

Reply:
Sure, I'm fine with switching it all later.